package-lock.json

```diff
@@ -2229,6 +2049,12 @@
             "resolved": "https://registry.npmjs.org/diagnostic-channel-publishers/-/diagnostic-channel-publishers-0.2.1.tgz",
             "integrity": "sha1-ji1geottef6IC1SLxYzGvrKIxPM="
         },
+        "didyoumean": {
+            "version": "1.2.1",
+            "resolved": "https://registry.npmjs.org/didyoumean/-/didyoumean-1.2.1.tgz",
+            "integrity": "sha1-6S7f2tplN9SE1zwBcv0eugxJdv8=",
+            "dev": true
+        },
         "diff": {
             "version": "3.3.1",

```

Comment:
Doesn't this have to be >= 3.5.0 to fix the issue?

Reply:
You make a good point. This was the result of `npm audit fix` which seemed to only update vsce.

Reply:
Ah I see, i was fixing a different advisory. There isn't an easy fix for the diff one.

I updated the PR description with the actual issue, and I will need to open a new PR to fix this diff one. 

Sorry about that.